plugins/web/opentelemetry-plugin-react-load/src/enums/AttributeNames.ts

```diff
@@ -16,6 +16,8 @@
 
 export enum AttributeNames {
   MOUNTING_SPAN = 'reactLoad: mounting',
+  UPDATING_SPAN = 'reactLoad: updating',
   LOCATION_URL = 'location',
-  REACT_NAME = 'react component name'
+  REACT_NAME = 'react component name',
+  REACT_STATE = 'react component state'
 }
```

Comment:
Having names with spaces in JSON payloads can be awkward and I believe the attribute names could be open to discussion. I didn't find documentation for open telemetry, but I know the attribute names that aren't custom are / or . delimited hierarchical names (ie. https://cloud.google.com/trace/docs/reference/v2/rest/v2/Attributes)

But since you've nicely written those names as constant values in an enum it shouldn't be a issue from a code point of view. It's more a product discussion point. Particularly I'd be interested in knowing if OpenTelemetry has a guideline for attribute names.

Reply:
It doesn't look like OpenTelemetry has an explicit documents commenting on attribute names, but other attribute enums use snake case in place of spaces so it makes sense to go with that instead (since it also seems like what the GCP docs look like)